javaparser-core/src/main/javacc/java_1_8.jj

```diff
@@ -25,11 +25,11 @@ options {
   JAVA_UNICODE_ESCAPE=true;
   COMMON_TOKEN_ACTION=true;
   JDK_VERSION = "1.8";
```

Comment:
How comes that's art of java_1_8.jj? I'd have assumed you'd create a new file java_9.jj instead.